dev/CalendarView/CalendarView_themeresources.xaml

```diff
@@ -456,9 +459,35 @@
                                     <ColumnDefinition Width="*" />
                                     <ColumnDefinition Width="*" />
                                 </Grid.ColumnDefinitions>
-                                <Button x:Name="HeaderButton" Content="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=TemplateSettings.HeaderText}" IsEnabled="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=TemplateSettings.HasMoreViews}" Style="{StaticResource NavigationButtonStyle}" Foreground="{TemplateBinding Foreground}" Padding="12,0,0,0" HorizontalContentAlignment="Left" />
-                                <Button x:Name="PreviousButton" Grid.Column="1" Content="&#xE0E4;" FontFamily="{ThemeResource SymbolThemeFontFamily}" IsTabStop="True" Padding="1" Foreground="{TemplateBinding Foreground}" HorizontalContentAlignment="Center" IsEnabled="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=TemplateSettings.HasMoreContentBefore}" Style="{StaticResource NavigationButtonStyle}" />
-                                <Button x:Name="NextButton" Grid.Column="2" Content="&#xE0E5;" FontFamily="{ThemeResource SymbolThemeFontFamily}" IsTabStop="True" Padding="1" Foreground="{TemplateBinding Foreground}" HorizontalContentAlignment="Center" IsEnabled="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=TemplateSettings.HasMoreContentAfter}" Style="{StaticResource NavigationButtonStyle}" />
+                                <Button x:Name="HeaderButton"
+                                    Content="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=TemplateSettings.HeaderText}"
+                                    IsEnabled="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=TemplateSettings.HasMoreViews}"
+                                    Style="{StaticResource NavigationButtonStyle}"
+                                    Foreground="{TemplateBinding Foreground}"
+                                    Padding="12,0,0,0"
+                                    HorizontalContentAlignment="Left" />
+                                <Button x:Name="PreviousButton"
+                                    Grid.Column="1"
+                                    Content="&#xE0E4;"
+                                    FontFamily="{ThemeResource SymbolThemeFontFamily}"
+                                    IsTabStop="True"
+                                    Padding="1"
+                                    Foreground="{TemplateBinding Foreground}"
+                                    HorizontalContentAlignment="Center"
+                                    IsEnabled="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=TemplateSettings.HasMoreContentBefore}"
+                                    Style="{StaticResource NavigationButtonStyle}"
+                                    ToolTipService.ToolTip="{Binding ElementName=PreviousButton, Path=(AutomationProperties.Name)}" />
```

Comment:
I think we need the localized string value here. (see TabView)

Reply:
I assumed the UIA Name was already localized ([see the TabView's Add button](https://github.com/microsoft/microsoft-ui-xaml/blob/e90f6f8fa7b8283e0114856ce69891d908fc3eb4/dev/TabView/Strings/fr-FR/Resources.resw#L120) for example)? The CalendarView resource files are not yet part of WinUI 2 so I cannot check them. 

@ranjeshj Can you check in your internal code-base if they are localized?

If they are not, how should we proceed here? Would using `x:Uid` in Xaml Markup and new resource files (to be merged with the internal existing resource files) be an option?